doc/resources/CaseInsensitiveInputStream.js

```diff
@@ -39,8 +51,8 @@ CaseInsensitiveInputStream.prototype.release = function(marker) {
     return this._stream.release(marker);
 };
 
-CaseInsensitiveInputStream.prototype.seek = function(_index) {
-    return this._stream.getText(start, stop);
+CaseInsensitiveInputStream.prototype.seek = function(index) {
+    return this._stream.seek(index);
```

Comment:
There are a few functions (`reset`, `consumer`, `seek`) that don't need to be `return`ed from.

Reply:
Updated